src/parser/typeRegistry.js

```diff
@@ -52,7 +52,7 @@ var elementTypes = [
   './xsd/extension',
   './xsd/group',
   './xsd/import',
-  './xsd/include',
+  './xsd/Include',
```

Comment:
@raymondfeng the file name is not wrong, it has a capital "I" ; we can either change the file name or the reference to it. If you think the standard is to have file names with small case letters, I will do that